File: PYTHON/CLASE PYTHON ESTUDIAR/EJERCICIOS PROPUESTOS PYTHON TIPOS DE DATOS CONDICIONALES Y BUCLE/ejercicio11.py
#    11. Escribir un programa que pregunte al usuario su edad y muestre por pantalla todos
#    los años que ha cumplido (desde 1 hasta su edad)

# Introducimos en la variable edad la edad, esto lo hacemos con input

edad = int(input("Introduce tu edad: "))


# Usamos un bucle for para mostrar los años cumplidos, imprimiendo al final el contador,
# en el rango le decimos que empiece en 1 y que acabe en edad+1, porque si no le sumamos
# uno, el bucle finalizará una pasada antes

for contador in range(1, edad+1):
    print(contador)

File: JAVA/CODIGOJAVA/BOLETINES/Boletin2 Condicionales y Bucles/src/Ejercicio11.java
/* 11. Diseñar un programa que muestre el producto de los 10 primeros números impares. */
public class Ejercicio11 {
    public static void main(String[] args) {
    int contador = 1;
    int producto = 1;
    int impares = 1;
    while (contador <= 10) {
            producto = producto * impares;
            impares += 2;
        contador++;
    }
        System.out.println("El producto es: " + producto);
    }
}
